src/sagas/repository.ts

```diff
@@ -592,6 +592,66 @@ export function* getJobs({ payload }: any): any {
   }
 }
 
+/**
+ * Get a single result
+ */
+export function* getJobResult({ payload }: any): any {
+  const { id } = payload;
+  try {
+    const response = yield call(authGet, `/api/repository/v1/jobs/${id}`);
+    const jobStatus = response.data.job_status;
+    if (jobStatus !== Status.RUNNING) {
+      let resultResponse;
+      try {
+        resultResponse = yield call(authGet, `/api/repository/v1/jobs/${id}/result`);
+        yield put({
+          type: ActionTypes.GET_JOB_RESULT_SUCCESS,
+          payload: {
+            data: {
+              id,
+              jobResult: {
+                resultType: 'success',
+                result: resultResponse.data,
+              },
+            },
+          },
+        });
+      } catch (err: any) {
+        if (err.response) {
```

Comment:
@lu-c - Do you think we should make a ticket to clean this up in the service?  It seems like we've conflated job status and http status of a request for job status - and this is the byproduct.

Reply:
Sure, let me add a ticket!